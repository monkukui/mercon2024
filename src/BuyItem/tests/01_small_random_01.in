9
521153118 405618965 80
244902542 776821429 94
466138156 972785771 1
879557474 696310048 44
928949490 70939691 42
487460915 554005938 16
476023583 929450483 52
45323991 820056024 94
840898160 305062969 39
