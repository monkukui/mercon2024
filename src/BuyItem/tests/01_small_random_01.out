9
70939691 928949490
305062969 840898160
405618965 521153118
554005938 487460915
696310048 879557474
776821429 244902542
820056024 45323991
929450483 476023583
972785771 466138156
